scripts/us_census/pep/us_pep_sex/process.py

```diff
@@ -91,62 +104,60 @@ def _national_1900_1979(file_path: str) -> pd.DataFrame:
+            
+            df.columns = [
+                'Age', 'Total', 'Count_Person_Male', 'Count_Person_Female',
+                'White Total', 'White Male', 'White Female', 'NonWhite Total',
+                'NonWhite Male', 'NonWhite Female'
+            ]
+            df = df.drop(columns=df.columns.difference(['Count_Person_Male', 'Count_Person_Female']))
+            
         else:
             df = pd.read_csv(file_path,
                              thousands=',',
                              engine='python',
-                             skiprows=7,
+                             skiprows=6,
                              skipfooter=102,
                              header=None)
-        df.columns = [
-            'Age', 'Total', 'Count_Person_Male', 'Count_Person_Female',
-            'White Total', 'White Male', 'White Female', 'NonWhite Total',
-            'NonWhite Male', 'NonWhite Female'
-        ]
-        df = df.drop(columns=[
-            'Age', 'Total', 'White Total', 'White Male', 'White Female',
-            'NonWhite Total', 'NonWhite Male', 'NonWhite Female'
-        ])
-        df['Year'] = year
-        df.insert(0, 'geo_ID', 'country/USA', True)
-        df['Measurement_Method'] = 'dcAggregate/CensusPEPSurvey_PartialAggregate'
-    else:
-        df = pd.read_csv(file_path,
-                         thousands=',',
-                         engine='python',
-                         skiprows=6,
-                         skipfooter=102,
-                         header=None)
-        df.columns = [
-            'Age', 'Total', 'Count_Person_Male', 'Count_Person_Female',
-            'White Total', 'White Male', 'White Female', 'Black Total',
-            'Black Male', 'Black Female', 'OtherRace Total', 'OtherRace Male',
-            'OtherRace Female'
-        ]
-        # dropping unwanted columns
-        df = df.drop(columns=[
-            'Age', 'Total', 'White Total', 'White Male', 'White Female',
-            'Black Total', 'Black Male', 'Black Female', 'OtherRace Total',
-            'OtherRace Male', 'OtherRace Female'
-        ])
+            df.columns = [
+                'Age', 'Total', 'Count_Person_Male', 'Count_Person_Female',
+                'White Total', 'White Male', 'White Female', 'Black Total',
+                'Black Male', 'Black Female', 'OtherRace Total',
+                'OtherRace Male', 'OtherRace Female'
+            ]
+            
+            # dropping unwanted columns
+            df = df.drop(columns=df.columns.difference(['Count_Person_Male', 'Count_Person_Female']))
+            
         # adding geoid, year and measurement method
         df['Year'] = year
         df.insert(0, 'geo_ID', 'country/USA', True)
         df['Measurement_Method'] = 'dcAggregate/CensusPEPSurvey_PartialAggregate'
-    return df
+        return df
+    except Exception as e:
+        logging.fatal(f"Error processing the file {file_path}: {e}")
+    except Exception as e:
```

Comment:
multiple except block are there. Remove the duplicate ones.

Reply:
Removed.